apps/www/components/Questionnaire/TextInput/TextInput.js

```diff
@@ -38,7 +38,7 @@ const TextInput = (props) => {
   const [colorScheme] = useColorContext()
   const { t } = useTranslation()
 
-  const { label, onChange } = props
+  const { label = '', onChange } = props
```

Comment:
don't we need the `text=''` too?

Reply:
That was already done on L33. I merged the two lines now :)